frontend/src/components/sprintboard/board.jsx

```diff
@@ -1,46 +1,51 @@
-import {mockSprints} from '../../../tests/utils/fake.js';
+import _ from 'lodash';
 import React from 'react/addons';
-// import Reflux from 'reflux';
-// import Actions from '../../actions/actions';
+import Reflux from 'reflux';
+import {SprintActions} from '../../actions/actions';
+import SprintStore from '../../stores/sprintStore';
 
-import SprintColumn from './column.jsx';
+import { DragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd/modules/backends/HTML5';
 
-let sprint = mockSprints(1, 3)[0];
+import SprintColumn from './column.jsx';
 
 let SprintBoard = React.createClass({
-  // mixins: [
-  //   Reflux.listenTo(HelloStore, 'onStoreUpdate')
-  // ],
+  mixins: [
+    Reflux.ListenerMixin
+  ],
 
   getInitialState() {
     return {
-      sprint: sprint,
-      sprintColumns: ['Backlog', 'Ready', 'In Progress', 'Done']
+      sprint: {},
+      sprintColumns: ['To Do', 'In Progress', 'Review', 'Done']
     };
   },
 
-  // onStoreUpdate(addressee) {
-  //   this.setState({
-  //     addressee: addressee
-  //   });
-  // },
+  componentDidMount() {
+    this.listenTo(SprintStore, this.onStoreUpdate);
```

Comment:
I was very curious about that too. The only way I could think of was like you are describing: trigger an event, pass a payload, and then in a function inside the component, depending on what the payload is, execute other appropriate functions.

But it feels like the idea is that a store provides a component with all the information it needs to know in one single message.

http://stackoverflow.com/questions/29048164/can-refluxjs-stores-indicate-which-property-has-changed-when-they-call-trigger